setup.py

```diff
@@ -41,7 +41,7 @@
     "scmdata",
     "scipy",
     "pint",
-    "pandas==0.24.2",  # Until next release https://github.com/IAMconsortium/pyam/issues/245
+    "pandas>=0.24.2",
```

Comment:
```suggestion
    "pandas>=0.25",
```

can you also pin the pyam version too please?

Reply:
I just set it to `>=0.4.0`

Reply:
Pandas needs to be pinned too right? Or is that pinning done by pyam?

Reply:
If it’s done by pyam we can delete the pandas pinning then right?

Reply:
The thing is in just released pyam (0.4.0) there is wrong pandas requirement (it requires >= 0.25 to pass all tests). I [updated](https://github.com/IAMconsortium/pyam/commit/957af4b08a567ebf9e960d3822db160af1a4783e) it yesterday but it was after release 0.4.0.

Reply:
Ah ok cool perfect then